api/boilerplate/views.py

```diff
@@ -17,3 +27,50 @@ class GroupViewSet(viewsets.ModelViewSet):
     """
     queryset = Group.objects.all()
     serializer_class = GroupSerializer
+    
+def open_id_auth(request):
+
+    redirect_to='https://eid.local/api/eIdService/init'
+    params = None
+    if request.method == "GET":
+        params = request.GET
+    elif request.method == "POST":
+        params = request.POST
```

Comment:
Does the openId Protocol not specify if the request has to be made by POST or GET?

Reply:
> Authorization Servers MUST support the use of the HTTP GET and POST methods defined in RFC 2616 [RFC2616] at the Authorization Endpoint. Clients MAY use the HTTP GET or POST methods to send the Authorization Request to the Authorization Server. If using the HTTP GET method, the request parameters are serialized using URI Query String Serialization, per Section 13.1. If using the HTTP POST method, the request parameters are serialized using Form Serialization, per Section 13.2.

taken from section 3.1.2.1 of [The OpenID Connect Core 1.0 Spec](http://openid.net/specs/openid-connect-core-1_0.html#AuthRequest)

Reply:
OK, great.